File: PHP/fizzbuzz.php
<?php
function fizzBuzz(int $n) {
    for ($i = 1; $i <= $n; $i++) {
        if ($i % 15 === 0) {
            echo "FizzBuzz\n";
        } elseif ($i % 3 === 0) {
            echo "Fizz\n";
        } elseif ($i % 5 === 0) {
            echo "Buzz\n";
        } else {
            echo "$i\n";
        }
    }
}

fizzBuzz(15);

File: Erlang/fizzbuzz.erl
% erl fizzbuzz.erl
% c(fizzbuzz).
% fizzbuzz:run().

-module(fizzbuzz).
-export([run/0]).

fizzbuzz(16) -> ok;
fizzbuzz(Count) ->
    if
      Count rem 15 == 0 -> io:fwrite("FizzBuzz\n");
      Count rem 3 == 0 -> io:fwrite("Fizz\n");
      Count rem 5 == 0 -> io:fwrite("Buzz\n");
      true -> io:format("~p~n", [Count])
    end,
    fizzbuzz(Count + 1).

run() ->
    fizzbuzz(1).
